keyboards/jj40/keymaps/stevexyz/keymap.c

```diff
@@ -261,6 +271,11 @@ void matrix_scan_user(void) {
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
+    case CK_CONFIGINIT:
+      if (record->event.pressed) {
+        eeconfig_init(); // reset keyboard to a standard default state; useful when new releases messup with eeprom values
```

Comment:
You don't need this, actually.  

There is the `EEP_RST` keycode which does this.

Reply:
In my opinion such a thing as a keyboard reset is more clear with a function call.
Anyway, since you're the maintainer, if you want I can change it, let me know or even better directly propose the code change to accept (and surely I will)...

Reply:
Well, you can can add a define at the top. 
eg, something like
```c
#define CK_CONFIGINIT EEP_RST
```

This would let you use whatever name you want, and still refer to the existing keycode.

Reply:
I agree here, having duplicate logic is just going to cause more issues in the long run. The keycode alias is fine though.

Reply:
changes committed